plugins/codemirror/plugin.js

```diff
@@ -51,11 +56,11 @@
 			});
 		},
 
-		_handleCodeMirrorChange: function (editor, oldData) {
+		_handleCodeMirrorChange: function (editor, editable, oldData) {
```

Comment:
Can we get rid of the unused `editor` parameter now?

Reply:
Yes, of course, I didn't pay attention to that.